contrib/ohi-release-notes/run.sh

```diff
@@ -110,7 +114,7 @@ fi
 
 
     # generating the changelog
-    ${RT_BIN} generate-yaml --excluded-dirs "$EXCLUDED_DIRECTORIES"
+    ${RT_BIN} generate-yaml --excluded-dirs "$EXCLUDED_DIRECTORIES" --excluded-files "$EXCLUDED_FILES"
```

Comment:
With the change that we are, we must have the use case of the excluded files and/or directories flags being empty.

```suggestion
    ${RT_BIN} generate-yaml "$EXCLUDED_DIRECTORIES_FLAG" "$EXCLUDED_FILES_FLAG"
```

Reply:
I do not thinks it is an issue:
```
rt generate-yaml --included-dirs --excluded-dirs
INFO[0000] Ignoring 0.0.2 since it belongs to a different branch 
INFO[0000] Renovate source did not find any commit since "190097158a3dda47531b636f81392f587d233faf" 
INFO[0001] Ignoring 0.0.2 since it belongs to a different branch 
INFO[0001] Dependabot source did not find any commit since "190097158a3dda47531b636f81392f587d233faf" 
changelog is empty

```

Reply:
Hhmm... I have a question then. Shouldn't then be mutual exclusive flags?

If we set `include` we should only have into account commits that affect that file. What is the use case to have both?

Reply:
Not sure if I understood the question:
You could have something like exclude dir "folder1" and exclude file "folder2/file"

Reply:
Take this as an example:
```
.github
go.mod
bundle.yaml
```

If I include `bundle.yaml` it makes no sense to "exclude `.github`" because we are excluding everything except `bundle.yaml`, right?

That is why I am asking if these are mutually exclusive. We should not allow using `exclude` and `include` flags at the same time.

Reply:
Since one is an allowList and the other is a denyList I think that the use-case to have them at the same time if whenever you have conflicts:
```
/allow-this
/allow-this/without-this
```

In the case above you can allow everything in `/allow-this` but not in `/allow-this/without-this`.

Reply:
When allowing/not-allowing files only I think you are right, it is not useful to have both options at the same time.

Reply:
The problem then is even worse. Which is the order?
What if I want to exclude a directory and include s ingle file of that folder?

From my point of view, it is better to just disallow to use both of the flags at the same time and take time to implement a good ignore mechanism like `.gitignore`, `.helmignore`, or `.dockerignore`.

Reply:
> take time to implement a good ignore mechanism
I fear that doing it would be a total overkill given our needs (Honestly, I would not have the include option at all and keep it simple)

Regarding the "order", it is specified in the comments: since we are defining filters, `exclude` takes precedence right 
now. If it is included and excluded at the same time -> it is excluded

Basically, a file is included if:
 - (it is not excluded) &&  (if any include rule is specified - it is included)

Reply:
I can document it in the `generate-yaml` readme if you think that having both is not that un-common

Reply:
documented!